hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyDeleteRequest.java

```diff
@@ -197,4 +198,118 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
     return omClientResponse;
   }
 
+  public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
```

Comment:
Can we avoid copying the same method here? We can call the new method with null bucket layout in the above method.

Reply:
Done